iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/TerminalUtil.java

```diff
@@ -0,0 +1,42 @@
+package com.powsybl.iidm.network.util;
+
+import com.powsybl.iidm.network.*;
+
+import java.util.Optional;
+
+public final class TerminalUtil {
```

Comment:
Could be moved to Terminal ?

Reply:
Don't know what you mean ? The whole class ? Terminal is only an interface with no implementation. I think having the helper function separated is good.

Reply:
yes, since Java 8 we can have static methods in interface. So before we use to have like `Terminals` ou `TerminalUtil` utility classes. Now (see JDK itself) we often see utility methods directly has a static method in the interface itself.